shared/common-adapters/button.desktop.js

```diff
@@ -52,6 +52,10 @@ class Button extends Component<void, Props, void> {
           opacity: disabled ? stylesButtonDanger.disabledOpacity : 1,
         }
         break
+      case 'Custom':
```

Comment:
I need a "style-less" button so I can set custom styles... And I don't have to undo defaults.

Should I make a "More" button here? Not sure how to proceed, need a suggestion.